### PR TITLE
docs(uppercase): Added an example

### PR DESCRIPTION
**Pull request by snood1205**
I saw that the uppercase filter had no example so I decided to add a minimal example to explain how the uppercase filter works.

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Docs update.

**What is the current behavior? (You can also link to an open issue here)**
There is currently no example for the `uppercase` filter, so I added one.

**What is the new behavior (if this is a feature change)?**
N/a

**Does this PR introduce a breaking change?**
No.

**Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

**Comment by Narretz** [comment]
Thanks for the PR! Can you please make these 2 changes:
- remove the protractor test (it's a very simple test, and we run these tests for every build, so it adds up)
- add a link to the uppercase filter that you can find an example in the lowercase filter docs?

**Comment by snood1205** [comment]
I remove the protractor file and added `@see angular.uppercase` to the docs for `lowercase`.

**Comment by snood1205** [comment]
@Narretz is this appropriate on the whole or would you like me to change this?

File: src/ng/filter/filters.js [hunk 1]
```diff
@@ -709,6 +710,22 @@ var lowercaseFilter = valueFn(lowercase);
  * @kind function
  * @description
  * Converts string to uppercase.
- * @see angular.uppercase
+ * @example
+   <example name="filter-uppercase">
```
Comment (Narretz) [inline src/ng/filter/filters.js]: you must add `module=uppercaseFilterExample`, otherwise the example won't work

File: src/ng/filter/filters.js [hunk 2]
```diff
@@ -699,6 +699,7 @@ function jsonFilter() {
  * @description
  * Converts string to lowercase.
  * @see angular.lowercase
+ * @see angular.uppercase
```
Comment (Narretz) [inline src/ng/filter/filters.js]: The docs don't actually do anything with `@see` :(, please revert this change

File: src/ng/filter/filters.js [hunk 3]
```diff
@@ -699,6 +699,7 @@ function jsonFilter() {
  * @description
  * Converts string to lowercase.
```
Comment (Narretz) [inline src/ng/filter/filters.js]: Please add a link after this that points to uppercase: Format is `{@link ng.filter.uppercase linktext}`

**Review by Narretz — CHANGES_REQUESTED** [review]

**Comment by Narretz** [comment]
Thanks - I fixed the link myself because I gave you the wrong path ;)

**Pull request merged**